docs/source/release_notes.rst

```diff
@@ -7,6 +7,7 @@ Future Release
 ==============
     * Enhancements
     * Fixes
+        * Fixed datetime inference function (:pr:`1413`)
```

Comment:
Minor nitpick, but this is actually fixing the `col_is_datetime` function which is used for inference, but also other places. Might be better to say `Fixed col_is_datetime utility function for correct handling of floats` or something similar here for clarity.